packages/react-theme/src/utils/mergeThemes.test.ts

```diff
@@ -1,21 +0,0 @@
-import { teamsLightTheme } from '../themes';
-import { mergeThemes } from './mergeThemes';
-import type { PartialTheme, Theme } from '../types';
-
-describe('mergeThemes', () => {
```

Comment:
are we going to loose these tests or is the plan to test the merge as part of `FluentProvider` tests?

Reply:
ha, so actually there is a test already covering this ;) so disregard 

![2022-01-13 at 18 27](https://user-images.githubusercontent.com/1223799/149379279-8baddea4-28ea-4702-bb7b-2b486a4dc3d2.png)